ga4gh/datamodel/datasets.py

```diff
@@ -251,49 +273,51 @@ class FileSystemDataset(AbstractDataset):
     def __init__(self, localId, dataDir, dataRepository):
         super(FileSystemDataset, self).__init__(localId)
         self._dataDir = dataDir
-        self._setMetadata()
 
         # Variants
         variantSetDir = os.path.join(dataDir, self.variantsDirName)
+        # We need a referenceSet instance for variants and features.
+        # To make this work we just pick the first reference set.
+        # This is NOT a good idea!!
```

Comment:
The GA4GH dataset is an arbitrary container, we shouldn't put
restriction on it.

Maciek Smuga-Otto <notifications@github.com> writes:
> In ga4gh/datamodel/datasets.py:
> 
> >
> >          # Variants
> >          variantSetDir = os.path.join(dataDir, self.variantsDirName)
> > +        # We need a referenceSet instance for variants and features.
> > +        # To make this work we just pick the first reference set.
> > +        # This is NOT a good idea!!
> 
> Question for @diekhans and others knowledgeable in the use of computational
> genomics tools: Would it be reasonable to assume that any given dataset is
> associated with precisely one reference set? If so, this association could be
> established in the dataset's metadata, and the rest of the code here would be
> pretty simple.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub*
>